sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1172,6 +1172,10 @@ def _handle_temp_and_staging_locations(self, validator):
       return []
     # Both staging and temp locations are bad, try to use default bucket.
     else:
+      _LOGGER.warning(
```

Comment:
It is valid when error is "ERR_MISSING_GCS_PATH". We should only issue warning when either is "ERR_MISSING_GCS_PATH"